lib/monitoring/aws-kinesisanalytics/KinesisDataAnalyticsMonitoring.ts

```diff
@@ -197,13 +233,17 @@ export class KinesisDataAnalyticsMonitoring extends Monitoring {
     });
   }
 
-  createNumberOfFailedCheckpointsWidget(width: number, height: number) {
+  createCheckpointFailuresWidget(width: number, height: number) {
```

Comment:
This method name change is technically a breaking change since we intentionally keep them public so the class is reasonably easy to extend if desired.

If we really want the new name that makes more sense, we should at least retain another deprecated method with the old name that just ends up calling the new one.